test/AWS.Deploy.CLI.IntegrationTests/AWS.Deploy.CLI.IntegrationTests.csproj

```diff
@@ -8,6 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="AWSSDK.CloudWatchLogs" Version="3.7.101.7" />
+    <PackageReference Include="AWSSDK.ECS" Version="3.7.105.4" />
```

Comment:
Could you please update the attribution document to this new version of ECS? or use the version we are already referencing? `3.7.100.14`

Reply:
Reverted to use the same version we are using the deploy tool itself `3.7.100.14`